src/fileUploadMeta/index.js

```diff
@@ -14,7 +14,8 @@ export default function getFileUploadMeta(kwargs = {}) {
       timestamp = Date.now(),
       extension = '.zip',
       auth: authorization,
-      method = 'POST'
+      method = 'POST',
+      networkTimeout = 5000
```

Comment:
Can this be overriden by an object passed from a plugin's initialization?

Reply:
Should do: see iopipe/iopipe-js-profiler#62